Add trefle data

LeiaQuesada opened the pull request:
https://trello.com/c/OEJHafiY/21-zoning-vegetation

Copy data from previously generated Trefle data archive at [treflehq](https://github.com/treflehq/dump/releases/tag/1.0.0-alpha%2B20201015)
Due to unexpected and abrupt shut down of their API service.

LeiaQuesada committed:
Add trefle data

Copy data from previously generated trefle data archive.
Due to unexpected and abrupt shut down of their API service.

LeiaQuesada committed:
Add compressed plant data

Work around GitHub file size limitations.

yjwangit reviewed (approved):
Nice job to add the license.md and readme.md files!

LeiaQuesada committed:
Refactor compressed plant data.

Save edible plants only, copy command to inject fields into newly created table. Consolidate sql creation and data to seed.pgsql.

sunnybrie reviewed (commented):
Your seed looks good! (All that plant data... 🥰  )I think you can merge this. If you haven't already, I recommend you check out the instructions I added to [George's deployment documentation](https://github.com/sunnybrie/express-react-project-example/blob/main/docs/deployment.md), particularly about making sure Heroku is linked properly to this database and not the example one you started with.
